### ISSUE TITLE
[BUG] All Citizen are not moving, hovering and sometimes random teleporting.

### ISSUE THREAD

**Issue by Knaxia** [ldtteam/minecolonies#9033]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Verison?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### Minecraft Version

1.19.3

### MineColonies Version

minecolonies-1.19.4-1.0.1321-ALPHA

### Structurize Version

structurize-1.19.4-1.0.489-ALPHA

### Related Mods and their Versions

(I can't select Minecraft 1.19.4 in the version above, but that's the version I'm running)

1. forge-1.19.4-45.0.31
2. blockui-1.19.4-0.0.73-ALPHA
3. domum_ornamentum-1.19.4-1.0.88-ALPHA-universal
4. journeymap-1.19.4-5.9.4beta2-forge
5. multipiston-1.19.3-1.2.26-ALPHA
6. OptiFine_1.19.4_HD_U_I3_MOD (Also tested without)
7. worldedit-mod-7.2.14

### Current Behavior

I decided to restart from scratch, made a new server, downloaded all the latest 1.19.4 mods, excluding JEI who isn't updated to 1.19.4. After creating my colony as usual, when the first people started to spawn, they appeared hovering in mid air, they don't seem to want to move and bumping into them does nothing. I tried giving them a floor under, or access to the townhall block area, but same time, if I recall them from the townhall, they will appear next to it, but again, not moving, can't bump them and if I remove the block under them, they'll keep hovering.

I made the builder hut, assigned a builder, but same thing with it, I had to force teleport and it started working, but when night came, didn't move, then slowly teleported bit by bit towards the town hall.

Think it might be the server, I tried in single player, disabled optifine (as suggested on discord) and same issue occur. I tried to downgrade to 1.0.1319 , but no go either.

I didn't have any issue in 1.19.3 before starting back from scratch on 1.19.4. All other normal minecraft mobs are acting normal.

Nothing in logs.

### Expected Behavior

NPCs should work normally...

### Reproduction Steps

1. Create new game
2. build town hall
3. Claim colony
4. wait for NPCs to spawn

### Logs

[latest.log](https://github.com/ldtteam/minecolonies/files/11106897/latest.log)

### Anything else?

Server:
![2023-03-29_21 07 47](https://user-images.githubusercontent.com/73440864/228704105-a8251777-eaba-4c12-bbb6-fa2ba0762284.png)

Single Player:
![2023-03-29_21 14 12](https://user-images.githubusercontent.com/73440864/228704141-55cbf254-09dc-4a7c-922e-73f555cf364b.png)

**Comment by Raycoms** [comment]
Did you try to use just minecolonies without other mods already?

-------- Original Message --------

**Comment by someaddons** [comment]
most likely optifine causing it

**Comment by Talyda** [comment]
I had them test without optifine, and it was still happening

**Comment by Raycoms** [comment]
Did they already try with a newer forge?

**Comment by Knaxia** [comment]
-I've had already disabled Optifine as written. Original test were done with a private dedicated server, which of course doesn't have it. It was only running on my client, which I disabled without any changes.
 -I was already running the latest forge at that moment too, can be seen in the logs. I had .29 but they had updated an hour after installing so I did update to .31 with same result, both server and client, before testing in single player mode.
-The only non dependency mods running were Journeymap and WorldEdit. All the mods listed are the only ones I'm using as I love the vanilla feel.

I can try again tonight without Journey and WE with a fresh installation again, and see if there's any forge update by then (and other mods).
